src/Item/utils.ts

```diff
@@ -25,3 +30,27 @@ export function toDBItem(item: FullItem): ItemAttributes {
     'content_hash',
   ])
 }
+
+export function buildTPItemURN(
+  thirdPartyId: string,
+  collectionURNSuffix: string,
+  itemURNSuffix: string
+): string {
+  return `${thirdPartyId}:${collectionURNSuffix}:${itemURNSuffix}`
+}
+
+// TODO: see if we need to use it
+export function decodeTPItemURN(
+  urn: string
+): { collectionId: string; itemId: string } {
+  const matches = tpwItemURNRegex.exec(urn)
+  if (matches === null || matches.length !== 2) {
+    throw new Error('The given collection URN is not TWP compliant')
+  }
+
+  return { collectionId: matches[0], itemId: matches[1] }
+}
+
+export function isTPItem(item: ItemAttributes): boolean {
+  return item.urn_suffix !== null
```

Comment:
Can/Should we use this on `isOwnedOrManagedBy` on `ItemService`?
```ts
} else if (dbItem.urn_suffix && dbItem.collection_id) {
  return this.collectionService.isOwnedOrManagedBy(
    dbItem?.collection_id,
    ethAddress
  )
} else {
```

Reply:
We should, yes! Added it here.